### ISSUE TITLE
Support WSL?

### ISSUE THREAD

**Issue by davidanger** [dartraiden/NVIDIA-patcher#75]
I try P102 with this patch, it can't work with WSL.
Is Anyone try successfully?

**Comment by XizumiK** [comment]
No, but P102-100 works well on Ubuntu even with offical driver. 
![image](https://github.com/dartraiden/NVIDIA-patcher/assets/159881522/8bfcb0ef-585e-4d44-99c0-67e787209bac)

**Comment by davidanger** [comment]
Updated:
It can support WSL

**Comment by dartraiden** [comment]
Try to patch libnvwgf2umx.so manually

for P1XX:
07 1B 07 00 87 1B 07 00 C7 1B 07 00 07 1C 07 00 09 1C 07
FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07

for CMP:
09 1E 07 00 49 1E 07 00 BC 1E 07 00 FC 1E 07 00 0B 1F 07 00 81 20 07 00 82 20 07 00 83 20 07 00 C2 20 07 00 89 21 07 00 0D 22 07 00 4D 22 07 00 8A 24 07
FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07 00 FF FF 07
